### PR TITLE
Use ignoreAllInputs instead of multiple if(noActions) { return; }

### PR DESCRIPTION
**Pull request by IvoryDuke**
Small optimization to use the recently added ignoreAllInputs GameState function

### CODE REVIEW

File: src/SSVOpenHexagon/Core/MenuGame.cpp [hunk 1]
```diff
@@ -81,6 +81,19 @@ MenuGame::MenuGame(Steam::steam_manager& mSteamManager,
...
 
+        // Scenario one: actions are blocked cause a dialog box is opened
+        if(!dialogBox.empty())
+        {
+            if(!(--noActions))
+            {
+                dialogBox.clearDialogBox();
+                game.ignoreAllInputs(false);
+                hg::Joystick::ignoreAllPresses(false);
+            }
```
Comment (SuperV1234) [inline src/SSVOpenHexagon/Core/MenuGame.cpp:92]: Can you please extract this to a private function or lambda? It's repeated a few times

**Review by SuperV1234 — CHANGES_REQUESTED** [review]

**Comment by SuperV1234** [comment]
Also, @IvoryDuke, please rebase on top of the current `master` -- I did apply some changes before I saw this, unfortunately...

File: src/SSVOpenHexagon/Core/MenuGame.cpp (hunk 1 above)
Comment (IvoryDuke) [reply src/SSVOpenHexagon/Core/MenuGame.cpp:92]: Sure will do

**Comment by IvoryDuke** [comment]
Not a problem will take care of it.

**Comment by IvoryDuke** [comment]
@SuperV1234 All done

**Review by SuperV1234 — APPROVED** [review]

**Pull request merged**